model-optimizer/requirements_onnx.txt

```diff
@@ -1,4 +1,4 @@
-onnx>=1.8.1
+onnx>=1.8.1,<1.12
```

Comment:
Why don't we support 1.12 latest stable version?

Reply:
there are no onnx for python3.6 package on pypi. Also see https://github.com/openvinotoolkit/openvino/pull/11949

Reply:
You can put different upper bounds for different Pythons in reqs files. I see no reason to set up such upper-bound.
For 2022.1 there was no such limitation and someone can already use it in his environment. Such limitation is really strange.

Reply:
The ONNX version in the requirements.txt files needs to be aligned with the version which we support in ONNX FE. Otherwise we will get CI failures for unsupported ops/tests.

Reply:
To consider for the future: to put upper limiter for higher version of dependency like this, even if in the moment of OV release such version of dependency doesn't exists.